provider/bikeshed/src/units/data.rs

```diff
@@ -107,6 +112,31 @@ impl crate::IterableDataProviderCached<UnitsDisplayNameV1Marker> for DatagenProv
             ))
         }
 
+        fn fill_data_locales(
+            data_locales: &mut HashSet<DataIdentifierCow<'_>>,
+            langid: &LanguageIdentifier,
+            length: &str,
+            length_patterns: &BTreeMap<String, Patterns>,
+        ) -> Result<(), DataError> {
+            let quantities: Vec<_> = length_patterns
+                .keys()
+                .filter(|&key| {
+                    !key.starts_with(|c: char| c.is_ascii_digit())
+                        && !["per", "times", "power"]
+                            .iter()
+                            .any(|&prefix| key.starts_with(prefix))
+                })
```

Comment:
this whole filter can be removed, as the next filter is more strict

Reply:
done